practice/bio/site.css

```diff
@@ -0,0 +1,41 @@
+body {
+  color: magenta;
+}
+
+div:after {
+  background-image: url("johncena.jpg");
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center center;
+  opacity: 0.25;
+}
+
+.link {
+  color: red;
+}
+
+.quote {
```

Comment:
It's easier to select all elements by type rather than adding a custom class to all elements of a type.